xbmc/network/NetworkServices.cpp

```diff
@@ -1060,7 +1060,7 @@ bool CNetworkServices::StopZeroconf()
 
 bool CNetworkServices::ValidatePort(int port)
 {
-  if (port <= 0 || port > 65535)
+  if (port < 0 || port > 65535)
```

Comment:
should this be in public port range too?

Reply:
Maybe? I didn't think to limit it to that range as Kodi was already 1 to max.